MapboxNavigation/StyleManager.swift

```diff
@@ -96,7 +96,7 @@ open class StyleManager: NSObject {
         resetTimeOfDayTimer()
     }
     
-    func applyStyle() {
+    func applyStyle(_ newStyle: Style? = nil) {
```

Comment:
We should be able to avoid the special case for custom style below if we’d take a `StyleType` instead of a specific style here.

Reply:
@frederoni you're right 👍